_data/projects/lynkhidr.yml

```diff
@@ -0,0 +1,8 @@
+name: LynkHidr
+desc: Link hider & shortener
+site: https://lynkhidr.onrender.com
+tags:
+  - shortener
+upforgrabs:
+  name: help wanted
+  link: https://github.com/corruptzero-dev/LynkHidr/labels/help%20wanted
```

Comment:
🤔 this label does not have any open issues, so the project won't be displayed on the site by default - did you have any plans to create some tasks?